clio.tests/ApplicationInstallerTests.cs

```diff
@@ -44,6 +44,45 @@ public void RestartApplicationAfterInstallPackageInNet6() {
 	}
 
 
+	[Test]
+	public void InstallWithOptionsCallsInstallWithOptionsWhenFailOnWarningIsTrue() {
+		string packagePath = "T:\\TestClioPackage.gz";
+		FileSystem.AddFile(packagePath, new System.IO.Abstractions.TestingHelpers.MockFileData(new byte[0]));
+		EnvironmentSettings environmentSettings = new EnvironmentSettings();
+		var applicationClientFactory = Substitute.For<IApplicationClientFactory>();
```

Comment:
We prefer to use Concrete types over var.
Instead of `var applicationClientFactory = Substitute.For<IApplicationClientFactory>();`, i'd like to see `IApplicationClientFactory applicationClientFactory = Substitute.For<IApplicationClientFactory>();`